.env.sample

```diff
@@ -0,0 +1,16 @@
+# This is an example of a `.env` file. The value are the default
+# that are set if the `.env` doesn't exist.
+# Please note that if present, this `.env` file is read both
+# by the `docker-compose.yml` and `Makefile`.
+
+WP_MARIADB_IMAGE_VERSION=11.4.4
+WP_PHP_IMAGE_VERSION=2025-047
```

Comment:
Any chance we can scrape these out of `quay-its.epfl.ch` instead? At least add a TODO comment in this direction...

Reply:
Well, it's the case, both in [Makefile](https://github.com/epfl-si/wp-dev/blob/544a0f48a34789957dd48bb7ad87176a61c6dfb0/Makefile#L49) and in the [docker-compose.yml](https://github.com/epfl-si/wp-dev/blob/544a0f48a34789957dd48bb7ad87176a61c6dfb0/docker-compose.yml#L41).

Reply:
This is done in 121e3aa025996430d1f645a6bd38e1faa2695ab9, but:
- It would be better to authenticate with the user's token (e.g. the one saved with `docker login`)
- This does not solve the variable expansion in the `docker-compose.yml` file
It still a bit of an improvement though.